test/utils.go

```diff
@@ -85,6 +95,11 @@ func setupProwConfig(resourceType string) string {
 	return project
 }
 
+func resetProwConfig(project string) error {
+	ticker.Stop()
```

Comment:
You should probably wait for the goroutine to finish using a channel

Reply:
Added a quit channel.